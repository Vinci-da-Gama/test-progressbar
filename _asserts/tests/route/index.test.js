import React from 'react';
import { shallow } from 'enzyme';
// import { MemoryRouter } from 'react-router-dom';

import Route from '../../src/route';

describe('In route/index.js Test Routes: ', () => {
    let shallowWrapper;
    beforeEach(() => {
        shallowWrapper = shallow(<Route />);
    });

    it('should render route correctlly.', () => {
        expect(shallowWrapper).toMatchSnapshot();
    });
});
