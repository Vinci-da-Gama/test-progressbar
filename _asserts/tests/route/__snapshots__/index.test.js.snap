// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`In route/index.js Test Routes:  should render route correctlly. 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Switch>
`;
